### ISSUE TITLE
[FEATURE] Infinite Aura

### ISSUE THREAD

**Issue by ghost** [CCBlueX/LiquidBounce#527]
**Describe your feature request.**
Basically like tp hit but it's automatic

**Potential settings for it**
Entity's to attack
Amount of entity's to attack at once
Max range
CPS
Render (shows a line of where it's tping)

**Additional context**
It's very useful for killing lots of people on servers if they disable their anticheat or don't have one in the first place

Video showing it here: https://youtu.be/YQH2bZuCLBc

**Comment by ENDER1355** [comment]
hmmm, you can do that with setting killaura range into 2147483647, but yes, it would be good to add it, cuz you don't have to change the range again.

**Comment by ghost** [comment]
No cause servers don't allow you to hit past 6 blocks or so, u gotta tp for it to work

**Comment by ENDER1355** [comment]
when clicktp added

**Comment by ghost** [comment]
Yea but it would also need to move around obstacles, not just go straight towards the enemy

**Comment by ghost** [comment]
That's where AI comes to the rescue!

**Comment by ghost** [comment]
Not really so, more like an algorithm

**Comment by 1zun4secondary** [comment]
> That's where AI comes to the rescue!

I would not call a few if checks AI.

**Comment by ghost** [comment]
Ye

**Comment by ENDER1355** [comment]
> Yea but it would also need to move around obstacles, not just go straight towards the enemy

when fight bot added

**Comment by ghost** [comment]
That could just be baritone with auto gap, eat and killaura

**Comment by ENDER1355** [comment]
> That could just be baritone with auto gap, eat and killaura

no it goes to the enemy and also moves around obstacles, u never used wurst?

**Comment by ghost** [comment]
That’s why I said baritone

**Comment by ENDER1355** [comment]
> That’s why I said baritone

yes but you also said auto gap and autoeat, which is unrelated, it just moves to the enemy and moves around obstacles and hits the enemy.

**Comment by ghost** [comment]
That’s so it doesn’t eventually die

**Comment by ENDER1355** [comment]
it would be buggy, since servers dont allow you to get 6 blocks of reach and you have to send a tp packet, sending too much will disconnect you, and without it, you cant, however, with customizing the tphit code, it can work, also with AI, you can move around obstacles.

**Comment by ghost** [comment]
It's not even ai, cause it doesn't learn from what it did and like I said, it's for servers that don't have/ accidentally disabled their ac

**Comment by ghost** [comment]
> it would be buggy, since servers dont allow you to get 6 blocks of reach and you have to send a tp packet, sending too much will disconnect you, and without it, you cant, however, with customizing the tphit code, it can work, also with AI, you can move around obstacles.

don't worry about buggy modules. im sure they will find a way to make it work without any problems.

**Comment by KMatias123** [comment]
tfw people say think AI is the same as A*
